scripts/linux/buildBranch.sh

```diff
@@ -21,7 +21,7 @@ SRC_BIN_DIR=$ROOT_FOLDER/bin
 SRC_DOCKER_DIR=$ROOT_FOLDER/docker
 SRC_SCRIPTS_DIR=$ROOT_FOLDER/scripts
 SRC_STRESS_DIR=$ROOT_FOLDER/stress
-SRC_E2E_TEMPLATES_DIR=$ROOT_FOLDER/smoke/IotEdgeQuickstart/e2e_deployment_files
+SRC_E2E_TEMPLATES_DIR=$ROOT_FOLDER/e2e_deployment_files
```

Comment:
Does this change anything about how the build job is configured?

Reply:
Release configuration has to be updated to work with the new path for the deployment file